# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/url/a-element-xhtml.xhtml.ini
# Commit: 9ad299b5161b
# Full Hash: 9ad299b5161b0cedf852d56365310de499f94043
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2018-11-14 22:12:17
# Regressor Bug: 1495313
# File Overlap Count: 1
# Description:
#   Bug 1495313 - Allow empty username in URLs r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D11254
# ==============================================================================

diff -r 6293789f6e9c -r 9ad299b5161b testing/web-platform/meta/url/a-element-xhtml.xhtml.ini
--- a/testing/web-platform/meta/url/a-element-xhtml.xhtml.ini	Wed Nov 14 14:27:58 2018 +0000
+++ b/testing/web-platform/meta/url/a-element-xhtml.xhtml.ini	Wed Nov 14 15:24:21 2018 +0000
@@ -17,9 +17,6 @@
   [Parsing: <http::@c:29> against <http://example.org/foo/bar>]
     expected: FAIL
 
-  [Parsing: <http://::@c@d:2> against <http://example.org/foo/bar>]
-    expected: FAIL
-
   [Parsing: <http://foo.com:b@d/> against <http://example.org/foo/bar>]
     expected: FAIL
 
@@ -176,12 +173,6 @@
   [Parsing: <http::b@www.example.com> against <about:blank>]
     expected: FAIL
 
-  [Parsing: <http:/:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
-  [Parsing: <http://:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
   [Parsing: <http:/:@/www.example.com> against <about:blank>]
     expected: FAIL
 
@@ -224,9 +215,6 @@
   [Parsing: <http://@:www.example.com> against <about:blank>]
     expected: FAIL
 
-  [Parsing: <http://:@www.example.com> against <about:blank>]
-    expected: FAIL
-
   [Parsing: <file:..> against <http://www.example.com/test>]
     expected: FAIL
 
@@ -404,9 +392,6 @@
   [Parsing: <tel:1234567890> against <http://example.org/foo/bar>]
     expected: FAIL
 
-  [Parsing: <https://:@test> against <about:blank>]
-    expected: FAIL
-
   [Parsing: <non-special://test:@test/x> against <about:blank>]
     expected: FAIL
 